src/frontend/details/Details.js

```diff
@@ -1,20 +1,19 @@
 import React, { Component } from 'react';
-import movieData from '../movie-data'
+//import movieData from '../movie-data'
 import { Link, Redirect } from "react-router-dom";
 import './details.css'
+import noImage from '../no-image.svg'
 
 export default class StatefulDetails extends Component {
   constructor(props) {
     super(props);
-    //Must initialise state first
     this.state = {
       error: null,
       isLoaded: false,
-      shows: [],
       title: '',
       synopsis: 'Default Synopsis',
       image: 'Default Image',
-      data: movieData
+      data: [{title: 'new movie'}]
```

Comment:
Mmm is there any point of doing this? Why don't you just create an empty array instead?